components/script/dom/documentorshadowroot.rs

```diff
@@ -207,24 +210,42 @@ impl DocumentOrShadowRoot {
         // Step 5
         elements
     }
-
     // https://html.spec.whatwg.org/multipage/#dom-document-activeelement
```

Comment:
Let's take this chance to update doc comment.
```suggestion
    /// <https://html.spec.whatwg.org/multipage/#dom-document-activeelement>
```